libs/api/domains/document-provider/README.md

```diff
@@ -0,0 +1,7 @@
+# Document-Provider-API
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
Consider improving the README

Reply:
Added minor documentation... will add more when there's more meat on the bones